package.json

```diff
@@ -102,7 +102,7 @@
   },
   "files": [
     "/dash_core_components/*{.js,.map}",
-    "/lib/**"
+    "/lib/**/!(*.jl)"
```

Comment:
Can't exclude the Julia artifacts with `babel` since we're using `--copy-files` for non-`/(j|t)s(x)?/ files, excluding from npm package instead